content/guides/virtualizing/bhyve.md

```diff
@@ -0,0 +1,158 @@
+
+```sh
+ifconfig bridge0 addm re0
+```
+
+Lastly, you want to run the following:
+
+```sh
+ifconfig bridge0 up
+```
+
+Now, make your changes permanent by editing `/etc/sysctl.conf/` and appending the following:
+
+```sh
+sysctl net.link.tap.up_on_open=1
+sysctl net.inet.ip.forwarding=1
+```
+
+
+### Downloading the Haiku Anyboot image <a name="part_download">
+
+Now that you have made the necessary changes to your system, you can now obtain a Haiku anyboot image securely, which will be necessary for the creation of a Haiku virtual machine. You will need to install `wget`, in order to fetch the image file from the internet. Because of the fact that Haiku image files are packed alongside with a `ReadMe` file inside of a `.zip` archive, you will need the `unzip` package in order to extract the image file.
+
+```sh
+pkg install wget unzip
+```
+
+After installing the two packages, we will fetch the archive containing the anyboot image file and then unzip it. For the purposes of this tutorial, the download of the 64-bit `r1beta1` from a specific Haiku mirror located in *East Coast, Germany* will be demonstrated, but if you want to choose a different mirror or a different version, then please consult the website [here](https://www.haiku-os.org/get-haiku/) and replace the following link accordingly.
+
+```sh
+wget https://s3.wasabisys.com/haiku-release/r1beta1/haiku-r1beta1-x86_64-anyboot.zip 
+unzip haiku-r1beta1-x86_64-anyboot.zip
+```
+
+As mentioned earlier, following files will be extracted in your current working directory:
+- `haiku-release-anyboot.iso`
+- `ReadMe.md`
+
+We will need the `haiku-release-anyboot.iso` file later, in order to virtualize Haiku.
+
+### Installing and running Haiku from an Anyboot image <a name="part_iso"></a>
+
+Now that you have made the required configurations, as well as the networking configurations, you may want to make a new folder in order to keep things clean, and then move your Haiku anyboot image there.
+
+```sh
+mkdir haiku-vm
+mv haiku-release-anyboot.iso haiku-vm
+cd haiku-vm
+```
+
+Great! Now you have a dedicated directory to work on, create a virtual disk for your virtual machine.
+
+```sh
+# feel free to adjust the number 16G according to your needs
+truncate -s 16G haiku-guest.img
+```
+
+In order to simplify the process, we will use `vmrun.sh`, an example script which can be found in the `/usr/share/examples/bhyve` folder.
+
+```sh
+cp /usr/share/examples/bhyve/vmrun.sh .
+sh vmrun.sh -c1 -E -i -I haiku-release-anyboot.iso haiku
+```
+
+- `-c1` stands for the amount processor cores that will be allocated for the machine. In this scenario, we are using 1 core, which is a recommended option.
+- `-E` boots the the machine in UEFI mode.
+- `-i` forces the boot of the a
+nyboot image.
+- `-I` specifies the location of the image file. In this case, since we have placed `haiku-release-anyboot.iso` in our current working directory, so this should suffice
```

Comment:
```suggestion
- `-I` specifies the location of the image file. In this case, since we have placed `haiku-release-anyboot.iso` in our current working directory, the file name should suffice.
```